docs/instances/instances.md

```diff
@@ -0,0 +1,33 @@
+# Universal Resolver — Instances
+
+Various instances of the Universal Resolver exist that are hosted by DIF as well as the DIF member community.
+
+## DIF instances
+
+### https://dev.uniresolver.io — hosted by DIF
+
+This is the development instance, which runs the latest code-base of the Universal Resolver project.
+
+See [here](/docs/instances/development-instance.md) for more information about this instance.
+
+### https://resolver.identity.foundation — hosted by DIF
+
+This is an alternate name for the same development instance (see above). In the future, this may become a redirect service, or reverse proxy, or load balancer, or similar service that will forward requests to the DIF member instances listed below:
+
+## DIF member instances
+
+This section lists additional instances of the Universal Resolver, hosted by DIF members.
+
+These instances may differ in terms of performance, security hardening, supported DID methods, etc.
+
+### https://resolver.spherity.com/ — hosted by Spherity
+
+(this space can contain specific information about this instance of the Universal Resolver)
+
+### https://resolver.indicio.tech/ — hosted by Indicio Tech
+
+(this space can contain specific information about this instance of the Universal Resolver)
+
+### https://resolver.danubetech.com/ — hosted by Danube Tech
+
+(this space can contain specific information about this instance of the Universal Resolver)
```

Comment:
Here's another, to be hosted by @spruceid:
```suggestion
(this space can contain specific information about this instance of the Universal Resolver)

### https://resolver.spruceid.com/ — hosted by Spruce Systems

(this space can contain specific information about this instance of the Universal Resolver)
```
It's not yet live, and we might use a different domain name, but this could be a placeholder for now.